lib_dock.ks

```diff
@@ -211,12 +215,35 @@ function dockMatchVelocity {
   // Cancel velocity
   lock throttle to min(matchVel:mag / matchAccel, 1.0).
 
-  wait until matchVel:z <= 0 and matchVel:mag <= residual and (residual = 0 or matchVel:mag > residual * 0.8).
-
+// This make not much sense...
+//wait until matchVel:z <= 0 and matchVel:mag <= residual and (residual = 0 or matchVel:mag > residual * 0.8).
```

Comment:
I can't recall what the point of this was. Unless you notice some crazy behavior, probably safe to assume that your simplified version is good enough.